proprietary/tokens/build.js

```diff
@@ -122,9 +122,6 @@ defaultMode.buildAllPlatforms()
 
 modes.map((mode) => {
   const styleDictionary = new StyleDictionary({
-    log: {
-      verbosity: 'verbose',
-    },
```

Comment:
This was duplicate config that appeared unnecessary. Removed it in this branch somewhere in the end of November, when I thought it would be merged soon anyway 😀